python/py-ubjson/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ubjson
+version             0.16.1
+platforms           darwin
+license             Apache-2
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         Universal Binary JSON encoder/decoder
+long_description    ${description}
```

Comment:
Use:
```
long_description    {*}${description}
```
Joshua made a big commit fixing this in many existing ports but we shouldn't reintroduce the error in new ports.

Reply:
Yep, fixed.